guides/common/modules/proc_mirroring-remote-flatpak-repositories-to-project-products.adoc

```diff
@@ -5,17 +5,21 @@
 
 You can mirror a Flatpak repository from a Flatpak remote into an existing product in {Project} to make it available for content management and distribution.
 
-This action creates a new repository inside the product you selected. 
-You can now synchronize the repository to pull down its content. 
+This action creates a new repository inside the product you selected.
+You can now synchronize the repository to pull down its content.
 Flatpak repositories are container repositories and you can add them to content views like other container repositories.
 
-Flatpak applications require a corresponding runtime environment, which the Flatpak remote also provides. 
+Flatpak applications require a corresponding runtime environment, which the Flatpak remote also provides.
 To make the {EL} 10 Mozilla Firefox Flatpak available to a host, ensure that the host can access the matching `rhel10/flatpak-runtime` repository.
 
+.Prerequisites
+* Your {Project} account has the following permissions for Flatpak remotes: `create_flatpak_remotes`, `view_flatpak_remotes`, `edit_flatpak_remotes`, and `destroy_flatpak_remotes`.
+* Ensure that Flatpak runtime repositories are available to clients alongside application repositories, because Flatpak applications such as `rhel9/firefox-flatpak` depend on them for installation.
```

Comment:
```suggestion
* Ensure that Flatpak runtime repositories are available to clients alongside application repositories, because Flatpak applications, such as `rhel9/firefox-flatpak`, depend on the runtime for installation.
```
Let's avoid pronouns for clarity.